webapp/components/LoginForm/LoginForm.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div id="login-form">
+    <form @submit.prevent>
+      <label>
+        <input
+          id="input_email"
+          v-model="email"
+          placeholder="Please type your E-Mail here"
+        />
+        <input
+          id="input_password"
+          v-model="password"
+          placeholder="Please type your password here"
+        />
+        <button
+          id="button_login"
+          @click="$emit('login', { email: email, password: password })"
+        >
+          Login
+        </button>
+      </label>
+    </form>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'LoginForm',
```

Comment:
⭐ ⭐ For a login feature in your webapp including a Vue component and its software tests.